files/en-us/mdn/structures/compatibility_tables/index.html

```diff
@@ -446,17 +450,38 @@ <h2 id="Pushing_a_change_back_to_the_main_repo">Pushing a change back to the mai
 
 <h2 id="Inserting_the_data_into_MDN_pages">Inserting the data into MDN pages</h2>
 
-<p>Once your new data has been included in the main repo, you can start dynamically generating browser compat tables based on that data on MDN pages using the {{TemplateLink("Compat")}} macro. This takes a single parameter, the dot notation required to walk down the JSON data and find the object representing the feature you want to generate the compat table for.</p>
+<p>Once your new compatibility data has been included in the main <a href="https://github.com/mdn/browser-compat-data">browser-compat-data</a> repo, you can start dynamically generating browser compatibility and specification tables based on that data on MDN pages.</p>
+
+<p>First you need to find the "address" for the relevant compatibility data you wish to include. This can be determined by inspecting its source file. For example:</p>
```

Comment:
I think we've been calling this query string or just query in other places.

```suggestion
<p>First you need to find the query string for the relevant compatibility data you wish to include. This can be determined by inspecting its source file. For example:</p>
```